frontend/www/js/omegaup/components/qualitynomination/List.vue

```diff
@@ -82,25 +116,36 @@ import { Vue, Component, Prop, Emit } from 'vue-property-decorator';
 import { omegaup } from '../../omegaup';
 import T from '../../lang';
 import * as UI from '../../ui';
-import paginador from '../common/Paginator.vue';
+import paginador from '../common/Paginatorv2.vue';
 import { types } from '../../api_types';
+import Autocomplete from '../Autocomplete.vue';
+import * as typeahead from '../../typeahead';
 
 @Component({
   components: {
     'omegaup-common-paginator': paginador,
+    'omegaup-autocomplete': Autocomplete,
   },
 })
 export default class QualityNominationList extends Vue {
   @Prop() pages!: number;
   @Prop() length!: number;
   @Prop() myView!: boolean;
-  @Prop() nominations!: omegaup.Nomination[];
-  @Prop() pagerItems!: types.PageItem[];
+  @Prop() nominations!: types.NominationList;
```

Comment:
Aquí se puede utilizar `types.NominationListItem[]` 
```suggestion
  @Prop() nominations!: types.NominationListItem[];
```

Reply:
listo